test/e2e/kuttl/inject-flagd/01-assert.yaml

```diff
@@ -1,3 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 5
+---
```

Comment:
Can you help me understand why this new (very small) resource is needed?

Reply:
I wanted to shorten the assertion time for the pod, but it can be removed